docs/pages/tutorial/eas/team-development.mdx

```diff
@@ -0,0 +1,184 @@
+---
+title: Develop with your team
+description: Learn how to use the EAS Update to send OTA updates and share previews with a team.
+---
+
+import { Step } from '~/ui/components/Step';
+import { Terminal } from '~/ui/components/Snippet';
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+import { Collapsible } from '~/ui/components/Collapsible';
+import Video from '~/components/plugins/Video';
+import { BoxLink } from '~/ui/components/BoxLink';
+import { BookOpen02Icon } from '@expo/styleguide-icons';
+
+Updates generally fix small bugs and push small changes in between app store releases. They allow updating the non-native parts of our app, such as JavaScript code, styling, and images.
+
+In this chapter, we're going to use [EAS Update](/eas-update/introduction/) to share changes with our team. This will help [us and our team quickly share previews](/eas-update/develop-faster/) of the change.
+
+<Step label="1">
+
+## Install expo-updates library
+
+To initialize our project and send an update, we need to use the [`expo-updates`](/versions/latest/sdk/updates/) library. Run the following command to install it:
+
+<Terminal cmd={['$ npx expo install expo-updates']} />
+
+</Step>
+
+<Step label="2">
+
+## Configure EAS Update
+
+To initialize our project with EAS Update, we need to:
+
+- Since we are using dynamic **app.config.js** for our app's configuration, adding [`updates`](/versions/latest/config/app/#updates) and [`runtimeVersion`](/eas-update/runtime-versions/#setting-runtimeversion) properties are required to make our project compatible with EAS Update. Run the following command to obtain these properties and their values from EAS and manually copy them to **app.config.js**:
+
+<Terminal cmd={['$ eas update:configure']} />
+
+<Collapsible summary="What about non-dynamic (app.json) projects?">
+
+If a project doesn't use dynamic app config (uses **app.json** instead of **app.config.js**), the above command will configure our app to be compatible with EAS Update and add the right properties to **app.json** and **eas.json**.
+
+</Collapsible>
+
+- Add a [`channel`](/eas/json/#channel) to every build profile in the **eas.json**. To automatically update this file, re-run the `eas update:configure` command:
+
+{/* prettier-ignore */}
+```json eas.json
+{
+  "build": {
+    "development": {
+      /* @hide ... */ /* @end */
+      "channel": "development"
+    },
+    "ios-simulator": {
+      /* @hide ... */ /* @end */
+      "channel": "ios-simulator"
```

Comment:
Shouldn't the channel be "development" here?

Reply:
I know you _can_, I just meany why would you have a separate channel for the iOS simulator, surely you'd just use the `development` channel.

Reply:
When there is an additional build profile in **eas.json** (apart from the default ones) and on running `eas update:configure`, it creates a channel for each profile based on the its name automatically. In a way, this is the default configuration based on the steps followed in the tutorial. I didn't originally thought of changing it. However, I see your point from app's development point of view. For example, my app is in development phase, I want to keep the development related config same for all environments.

I think I'll just update this in the tutorial and make a note for the reader to remember removing it before proceeding since the `ios-simulator` profile already extends `development` profile.

Reply:
Added the callout describing the above behaviour.